### ISSUE TITLE
pyqrcode in RhinoPython - Grasshopper

### ISSUE THREAD

**Issue by mnooner256** [mnooner256/pyqrcode#41]
I received the following report via email. I am moving the conversation over to github for posterity...

> Some days ago I tried to run pyqrcode from Rhino Python in my Rhino (64-bit) but i could not import the pyqrcode library. I tried to change the path but nothing changed. The message that i receive is:
> 
> Message: No module named pyqrcode
> 
> Traceback:
>   line 1, in <module>, "C:\Users\MEDINA\AppData\Local\Temp\TempScript.py"
> 
> How can I fix this issue?
> 
> Looking forward your response

**Comment by mnooner256** [comment]
I have never used Rhino, so I can only guess at the problem. In fact, I  have only vaguely heard of the program. Hence, I am going to try to help you via some _blind debugging._

The error you gave is almost always caused by the system not being able to find the pyqrcode directory where it looks for modules. Try this simple 2 line program:

```
import sys
print sys.path
```

The above program will print a list of paths that Python uses to find modules. Look for one with "site-packages" as part of the directory structure. The pyqrcode module folder should be put there.

If changing the folder's location does not help, please reply this comment with the exact location of the pyqrcode directory on your system and a list of the files pyqrcode directory. This will let me check that whether you have an installation problem or Rhino is doing something funky.

**Comment by ekevoo** [comment]
Maybe they don't know they need to `pip install` it first?

**Comment by mnooner256** [comment]
I looked over the Rhino Python primer. It does not really explain how to install third-party modules. My guess is a _by-hand_ installation was done. I ran similar problems in the past when I first started using python (way back before pip existed).

I am 99% sure it is an installation problem. But how the Rhino editor might be messing with path, is not something I can just advise him on a-priori. It might very well be that he installed it correctly, but Rhino does something funky to sys.path. Hence, the sanity check.
